package/MDAnalysis/analysis/msd.py

```diff
@@ -387,7 +387,10 @@ def _conclude_simple(self):
         """
         lagtimes = np.arange(1, self.n_frames)
         positions = self._position_array.astype(np.float64)
-        for lag in lagtimes:
+
+        # progressbar
+        from tqdm import tqdm
```

Comment:
If all paths lead to an import of tqdm, could you move this up to the top of the file please?

Reply:
> If all paths lead to an import of tqdm, could you move this up to the top of the file please?

Yeah, I'll make this change. Thank you!